test/functional/mempool_truc.py

```diff
@@ -458,21 +474,20 @@ def test_reorg_2child_rbf(self):
         node = self.nodes[0]
         self.log.info("Test that children of a TRUC transaction can be replaced individually, even if there are multiple due to reorg")
 
+        # Prep for fork
+        fork_blocks = create_empty_fork(node, fork_length=FORK_LENGTH)
         ancestor_tx = self.wallet.send_self_transfer_multi(from_node=node, num_outputs=2, version=3)
         self.check_mempool([ancestor_tx["txid"]])
 
-        block = self.generate(node, 1)[0]
+        self.generate(node, 1)[0]
         self.check_mempool([])
 
         child_1 = self.wallet.send_self_transfer(from_node=node, version=3, utxo_to_spend=ancestor_tx["new_utxos"][0])
         child_2 = self.wallet.send_self_transfer(from_node=node, version=3, utxo_to_spend=ancestor_tx["new_utxos"][1])
         self.check_mempool([child_1["txid"], child_2["txid"]])
 
-        self.generate(node, 1)
-        self.check_mempool([])
-
         # Create a reorg, causing ancestor_tx to exceed the 1-child limit
-        node.invalidateblock(block)
+        self.trigger_reorg(fork_blocks)
```

Comment:
this changes behavior a bit but should give the coverage we want still, as all we care about is getting 1p2c into mempool with TRUC